spec/API_specification/creation_functions.md

```diff
@@ -33,11 +33,11 @@ This function cannot guarantee that the interval does not include the `stop` val
 
 -   **step**: _Union\[ int, float ]_
 
-    -   the distance between two adjacent elements (`out[i+1] - out[i]`). Default: `1`.
+    -   the distance between two adjacent elements (`out[i+1] - out[i]`). Must not be `0`; may be negative, this results in an empty array if `stop <= start`. Default: `1`.
```

Comment:
Is this backwards? If the step is negative, then it's empty if stop >= start.

Reply:
yep, good catch, fixed